data/account/participationRegistry_test.go

```diff
@@ -1039,6 +1039,54 @@ func TestFlushResetsLastError(t *testing.T) {
 	a.NoError(registry.Flush(10 * time.Second))
 }
 
+// TestParticipationDB_Locking tries fetching StateProof keys from the DB while the Rolling table is being updated.
+// Makes sure the table is not locked for reading while a different one is locked for writing.
+func TestParticipationDB_Locking(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	a := require.New(t)
+
+	dbName := strings.Replace(t.Name(), "/", "_", -1)
+
+	dbfile, err := db.OpenErasablePair(dbName + ".sqlite3")
+	a.NoError(err)
+
+	registry, err := makeParticipationRegistry(dbfile, logging.TestingLog(t))
+	require.NoError(t, err)
+	require.NotNil(t, registry)
+
+	defer registryCloseTest(t, registry, dbName)
+
+	var id2 ParticipationID
+	for i := 0; i < 3; i++ {
+		part := makeTestParticipation(a, 1, 0, 511, config.Consensus[protocol.ConsensusCurrentVersion].DefaultKeyDilution)
+		id, err := registry.Insert(part)
+		if i == 0 {
+			id2 = id
+		}
+		a.NoError(err)
+		a.NoError(registry.AppendKeys(id, part.StateProofSecrets.GetAllKeys()))
+		a.NoError(registry.Flush(defaultTimeout))
+		a.Equal(id, part.ID())
+	}
+
+	go func() {
+		for i := 0; i < 100; i++ {
+			fmt.Printf("removing %d\n", i)
```

Comment:
I think we can remove the printfs here

Reply:
it could be t.Logf() if it's useful on failure, otherwise yes, remove `fmt.Printf` here and below